src/item.rs

```diff
@@ -437,7 +461,7 @@ pub fn write_timestamp<W: Write>(writer: &mut W, date_time: &DateTime<Utc>) -> R
 pub fn read_timestamp<R: Read>(reader: &mut R) -> Result<DateTime<Utc>> {
     let seconds = reader.read_le::<i64>()?;
     let nanos = reader.read_le::<u32>()?;
-    Ok(DateTime::<Utc>::from_utc(NaiveDateTime::from_timestamp(seconds, nanos), Utc))
```

Comment:
NaiveDateTime::from_timestamp is deprecated and from_timestamp_opt should be used.

Reply:
Hi, I a bit confused. I am not using from_timestamp_opt? 
Was this a self note/